spec/rubocop/cop/sketchup_requirements/initialise_entity_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::SketchupRequirements::InitialiseEntity do
```

Comment:
I'm sorry, I messed up last round of comments, the correct spelling in Ruby is `initialize`. (I don't know why I thought otherwise before.)